scripts/setup/interactive_env_setup.py

```
@@ -0,0 +1,409 @@
+        config["SENTRY_ENVIRONMENT"] = config["ENVIRONMENT"]
+
+    return config
+
+
+def write_env_file(config: dict[str, str], output_path: Path) -> None:
+    """Write configuration to .env file"""
+    lines = []
+
+    lines.append("# ============================================")
+    lines.append("# TOOLBOXAI ENVIRONMENT CONFIGURATION")
+    lines.append(f"# Generated by interactive setup")
```

Ruff (0.14.4)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)